package.json

```diff
@@ -52,6 +52,7 @@
     "aws-amplify": "6.2.0",
     "cropperjs": "1.6.2",
     "decimal.js": "10.4.3",
+    "framer-motion": "11.2.5",
```

Comment:
only use `ReOrder` component and related hooks, other comps will just be tree shaked